themes/default/layouts/learn/glossary/list.html

```diff
@@ -27,7 +26,7 @@ <h1>{{ .Title }}</h1>
             {{ range $terms }}
                 {{ $anchor := .term | anchorize }}
                 <div class="flex-1 mt-8">
-                    <h2 id="{{ $anchor }}" class="mt-2 mb-0">{{ .term }} </h2>
```

Comment:
Having prettier lint remove this extra space might introduce a subtle bug where that space is intentional. Depending on when we run prettier this could be confusing to someone without context on how prettier works. Though would like to hear what other folks think.

Reply:
can you give an example of where we are using an extra space here? or a situation you imagine we would want that in the future?